### DISCUSSION TITLE
Can't find MySQL as a script language

### DISCUSSION THREAD

**Discussion by hialvaro** [windmill-labs/windmill#1591] [Questions] [answered]
I've seen this merged pull request:
https://github.com/windmill-labs/windmill/pull/982

But can't seem to find it anywhere on the UI. Is this released?

**Comment by rubenfiszel** [comment, marked as answer]
We removed it as a template but can add it again. Under the hood it was just a template for using: https://deno.land/x/mysql@v2.11.0 which can still be done from the typescript scripts. Otherwise python libraries: https://github.com/PyMySQL/PyMySQL and deno seems to be also suggesting mysql2 from npm https://deno.com/manual@v1.28.3/node/how_to_with_npm/mysql2

**Reply by hialvaro** [reply]
Awesome, thanks.
